docs/es/docs/deployment/manually.md

```diff
@@ -19,13 +19,13 @@ You just need to install an ASGI compatible server like:
     </div>
 
     !!! tip
-        By adding the `standard`, Uvicorn will install and use some recommended extra dependencies.
+        Al añadir el `standard`, Uvicorn instalara y usara algunas dependencias extra recomendadas.
```

Comment:
A couple of accent marks are missing: `... instalará y usará algunas dependencias ...`